integration-tests/bats/sql-local-remote.bats

```diff
@@ -108,3 +108,29 @@ teardown() {
     [[ "$output" =~ "defaultDB does not exist" ]] || false
 }
 
+@test "sql-local-remote: verify dolt blame behavior is identical in switch between server/no server" {
+    cd defaultDB
+
+    dolt sql -q "create table test (pk int primary key)"
+    dolt sql -q "insert into test values (1)"
+    dolt add test
+    dolt commit -m "insert initial value into test"
+    dolt sql -q "insert into test values (2), (3)"
+    dolt add test
+    dolt commit -m "insert more values into test"
+
+    start_sql_server defaultDB
+    run dolt --user dolt blame test
+    [ "$status" -eq 0 ]
+    [[ "$output" =~  "1".*"insert initial value into test" ]] || false
+    [[ "$output" =~  "2".*"insert more values into test" ]] || false
+    [[ "$output" =~  "3".*"insert more values into test" ]] || false
+    stop_sql_server 1
+
+    run dolt blame test
+    [ "$status" -eq 0 ]
+    [[ "$output" =~  "1".*"insert initial value into test" ]] || false
+    [[ "$output" =~  "2".*"insert more values into test" ]] || false
+    [[ "$output" =~  "3".*"insert more values into test" ]] || false
```

Comment:
I think a simpler check would be to compare the total output of both blame calls. The line by line information will be the same if that checks out, and the tests as it stands now would allow for one implementation to drop the commit date and we'd never know it.